utopia-vscode-common/package.json

```diff
@@ -14,10 +14,10 @@
     "url": "https://github.com/concrete-utopia/utopia"
   },
   "scripts": {
-    "build": "webpack-cli",
-    "production": "webpack-cli --mode production",
-    "watch": "webpack-cli --mode production --watch --info-verbosity verbose",
-    "watch-dev": "webpack-cli --watch --info-verbosity verbose",
+    "build": "tsc",
+    "production": "tsc",
+    "watch": "tsc --watch --verbose",
```

Comment:
do we need thise --verbose here? what does it do? I can't find much info by googling for it